content/lessons/genesis-pt2.mdx

```diff
@@ -5,8 +5,8 @@ contentType: 'challenge'
 
 # Let's make sense of this
 
-Use the following command to decode Satoshi's secret HEX message into ASCII, a more human readable format:
+To decode Satoshi's secret hidden in the message you saw in the last exercise, use the following command and replace \<scriptSigHex\> with the message to interpret it.
 
 ```shell
-echo scriptSigHex | xxd -r -p
+echo <scriptSigHex> | xxd -r -p
```

Comment:
I totally agree. If someone could hover over the <scriptSigHex> portion and have a tool tip appear that said something like "Replace this text with the scriptSigHex you copied in the last challenge", it would be a lot clearer.